restore-sql-server-transaction-logs/scheduled-upload/upload-script.ps1

```diff
@@ -0,0 +1,112 @@
+# Default variables
+$uploadedObjectsNumber=0
+$currentDateTimeUtc = (Get-Date).ToUniversalTime()
+
+# Define the logic to set the content metadata the functions below. Exposed in functions for visibility and convenience.
+function Get-CloudSqlInstance-MetadataTag {
+  param ($file)
+
+  #Fill in with the destination Cloud SQL for SQL Server instance name
+  return ""
```

Comment:
Is this something that users need to provide? If so, why is not near the other settings below?

Reply:
Changed so that the CSQL instance name is an optional setting used in the function. It's either fixed, provided in the settings and the same for all files, or it can be inferred from the files name.

Reply:
LGTM